extensions/amp-bodymovin-animation/amp-bodymovin-animation.md

```diff
@@ -0,0 +1,72 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-bodymovin-animation"></a> `amp-bodymovin-animation`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>An <code>amp-bodymovin-animation</code> displays the AirBnB Bodymovin animation used to render the JSON generated by Adobe Effects.
```

Comment:
Let's  link to http://airbnb.io/lottie/ somewhere here.

Reply:
Done.